### PR TITLE
Add semi-colons to example std::any

### PR DESCRIPTION
**Pull request by alantsui4**
As stated in the title, thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -545,10 +545,10 @@ if (auto str = create(true)) {
 A type-safe container for single values of any type.
 ```c++
 std::any x {5};
-x.has_value() // == true
-std::any_cast<int>(x) // == 5
-std::any_cast<int&>(x) = 10;
-std::any_cast<int>(x) // == 10
+x.has_value(); // == true
+std::any_cast<int>(x); // == 5
+std::any_cast<int&>(x); = 10;
```
Comment (AnthonyCalandra) [inline README.md]: Syntax error here.

Comment (alantsui4) [reply README.md]: I have removed the syntax error in line 550

Comment (AnthonyCalandra) [reply README.md]: That line should be `std::any_cast<int&>(x) = 10;`. The left side casts `x` to an `int&` and assigns it the value `10`. It's like if you wrote it out like this:
```cpp
int& x_ref = std::any_cast<int&>(x);
x_ref = 10;
```
Also, I forgot to mention to please make the change to CPP17.md as well. Sorry about that!

Comment (alantsui4) [reply README.md]: The error is fixed

File: CPP17.md [hunk 2]
```diff
@@ -293,10 +293,10 @@ if (auto str = create(true)) {
 A type-safe container for single values of any type.
 ```c++
 std::any x {5};
-x.has_value() // == true
-std::any_cast<int>(x) // == 5
+x.has_value(); // == true
+std::any_cast<int>(x); // == 5
 std::any_cast<int&>(x) = 10;
-std::any_cast<int>(x) // == 10
+std::any_cast<int>(x) = 10;
```
Comment (AnthonyCalandra) [inline CPP17.md]: You had it correct before. Change this line back to `std::any_cast<int>(x); // == 10`

File: README.md [hunk 3]
```diff
@@ -545,10 +545,10 @@ if (auto str = create(true)) {
...
 ```c++
 std::any x {5};
-x.has_value() // == true
-std::any_cast<int>(x) // == 5
-std::any_cast<int&>(x) = 10;
-std::any_cast<int>(x) // == 10
+x.has_value(); // == true
+std::any_cast<int>(x); // == 5
+std::any_cast<int&>(x); //= 10
+std::any_cast<int>(x) = 10;
```
Comment (AnthonyCalandra) [inline README.md]: This example is broken now. It should look like this:
```cpp
std::any x {5};
x.has_value(); // == true
std::any_cast<int>(x); // == 5
std::any_cast<int&>(x) = 10;
std::any_cast<int>(x); // == 10
```

**Comment by AnthonyCalandra** [comment]
Any update? You're almost there! After addressing my two comments it will be good to merge.

**Comment by AnthonyCalandra** [comment]
Closed due to inactivity.

**Pull request closed without merging**